portfolio/src/main/webapp/script.js

```diff
@@ -16,120 +16,195 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-    const greetings = ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!','Salut :^)'];
```

Comment:
Generally speaking, it is better to not mix whitespace changing patches with topic changes.